addons/mass_mailing/wizard/mailing_mailing_test_views.xml

```diff
@@ -7,13 +7,13 @@
             <field name="arch" type="xml">
                 <form string="Send a Sample Mail">
                     <p class="text-muted">
-                        Send a sample email for testing purpose to the address below.
+                        Send a sample mailing for testing purpose to the address below.
                     </p>
                     <group>
-                        <field name="email_to"/>
+                        <field name="email_to" placeholder="email1@example.com&#10;email2@example.com"/>
```

Comment:
Shouldn't we have a classic "e.g. email1@example.com, .." ? Maybe with carriage returns it will be moche, not sure.

Reply:
We tested many different displays with LNA and this one was the voted best.